apps/condo/domains/resident/tasks/helpers/notifyOnPaydayisCurrnetDateValid.js

```diff
@@ -0,0 +1,13 @@
+const dayjs = require('dayjs')
```

Comment:
Spell in file name notifyOnPaydayisCu**rrnet**DateValid.js -> notifyOnPaydayIsCurrentDateValid.js